tasks.py

```diff
@@ -44,6 +44,30 @@ def build(context, netbox_ver=NETBOX_VER, python_ver=PYTHON_VER):
     )
 
 
+@task
+def generate_packages(context, netbox_ver=NETBOX_VER, python_ver=PYTHON_VER):
+    """Generate all Python packages inside docker and copy the file locally under dist/.
+
+    Args:
+        context (obj): Used to run specific commands
+        netbox_ver (str): NetBox version to use to build the container
+        python_ver (str): Will use the Python version docker image to build from
+    """
+    container_name = f"{BUILD_NAME}_netbox_package"
+    context.run(
+        f"docker rm {container_name} || true", env={"NETBOX_VER": netbox_ver, "PYTHON_VER": python_ver}, pty=True,
+    )
+    context.run(
+        f"docker-compose  -f {COMPOSE_FILE} -p {BUILD_NAME} run --name {container_name} -w /source netbox poetry build",
```

Comment:
Is this task needed, and does it really need to use docker? Can we not just run `poetry build` directly?

Reply:
perhaps not, I just wanted it to align with our other plugins and it's a handy method to have.  The nice part about doing this in docker is it doesn't require me to have poetry installed in my development environment.

Reply:
+1 here for `poetry build` directly